gsa/src/web/pages/targets/dialog.js

```diff
@@ -107,6 +118,10 @@ const NEW_SNMP = {
   types: SNMP_CREDENTIAL_TYPES,
 };
 
+const ElevatePrivilegeText = styled(Layout)`
+  color: ${Theme.darkRed};
```

Comment:
Actually, this should not be red. I talked about this with @bjoernricks and we realized that was an error in the mockup.

Reply:
Will just regular black work?

Reply:
yes just use the standard black.